CHANGELOG.md

```diff
@@ -19,6 +19,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 #### Changed
 
+- After opening => redact and send task and actions no longer explicitly mention
```

Comment:
I think this should go in the `Content` section ideally

Reply:
Yeah good shout, not sure why I forgot about the content section this time 🤷‍♀️ 😁